docs/ref/forms/fields.txt

```diff
@@ -66,7 +66,9 @@ an empty value -- either ``None`` or the empty string (``""``) -- then
     ...
     ValidationError: ['This field is required.']
     >>> f.clean(" ")
-    ' '
+    Traceback (most recent call last):
+    ...
+    ValidationError: ['This field is required.']
```

Comment:
This docs issue was introduced by `strip` being added to `CharField` 11cac1bd8ef7546ca32d9969d4348bf412dc6664
I'm not sure if we should keep this as an example here, we could remove it or reference strip or make these examples for an `IntegerField`
This also makes the above `an empty value -- either ``None`` or the empty string (``""``) -- then` less concrete

Reply:
All other changes have been pulled out to here: https://github.com/django/django/pull/18761/
Please update this PR to only solve this issue :+1:

Reply:
I think the above example should be removed because it is related to Field.required.
And if we add an example, it would be better to add it to the strip(`.. attribute:: strip`) part, but I don't think we need an example because I think it's rarely case allowed to entered only multiple blanks.

Reply:
Agreed :+1: